integration/experiment/gpu_frequency_sweep/gpu_frequency_sweep.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+'''
+Helper functions for running gpu frequency sweep experiments.
+'''
+
+import sys
+import argparse
+import os
+
+import geopmpy.agent
+
+from experiment import launch_util
+from experiment import common_args
+from experiment import machine
+from experiment.frequency_sweep import frequency_sweep
+from experiment.uncore_frequency_sweep import uncore_frequency_sweep
+
+def setup_run_args(parser):
+    common_args.setup_run_args(parser)
+    common_args.add_min_frequency(parser)
+    common_args.add_max_frequency(parser)
+    common_args.add_step_frequency(parser)
+    common_args.add_run_max_turbo(parser)
+    common_args.add_agent_list(parser)
+
+    parser.add_argument('--max-uncore-frequency', dest='max_uncore_frequency',
+                        action='store', type=float, default=None,
+                        help='top uncore frequency setting for the sweep')
+    parser.add_argument('--min-uncore-frequency', dest='min_uncore_frequency',
+                        action='store', type=float, default=None,
+                        help='bottom uncore frequency setting for the sweep')
+    parser.add_argument('--step-uncore-frequency', dest='step_uncore_frequency',
+                        action='store', type=float, default=None,
+                        help='increment in hertz between uncore frequency settings for the sweep')
+
+    parser.add_argument('--max-gpu-frequency', dest='max_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='top gpu frequency setting for the sweep')
+    parser.add_argument('--min-gpu-frequency', dest='min_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='bottom gpu frequency setting for the sweep')
+    parser.add_argument('--step-gpu-frequency', dest='step_gpu_frequency',
+                        action='store', type=float, default=None,
+                        help='increment in hertz between gpu frequency settings for the sweep')
+
+    parser.set_defaults(agent_list='frequency_map')
+
+
+def setup_gpu_frequency_bounds(mach, min_gpu_freq, max_gpu_freq,
```

Comment:
```suggestion
def setup_gpu_frequency_bounds(mach, min_gpu_freq=None, max_gpu_freq=None,
```
Use =None where appropriate to represent optional arg